### ISSUE TITLE
2016 and 2012 comparisons

### ISSUE THREAD

**Issue by code-issues-pull** [cjph8914/2020_benfords#3]
It would be interesting to see these same districts compared to the 2016 results and the 2012 results (where an incumbent was running).

**Comment by ghost** [comment]
Agreed, but remember that the distribution of precinct sizes is unlikely to be random. If precincts e.g. have a median of 9,000 eligible voters (with a certain distribution, e.g. Gaussian, and a certain standard deviation) then someone who quite consistently gains 60% of that vote should be expected to get around 5,000 votes in each precinct, again with a certain standard deviation. This calls the validity of applying Benford's Law to this particular case into question. So check the raw data to see if something like this could be the case. Normally precinct sizes are carefully planned and thus not random.

**Comment by chavenor** [comment]
https://results.enr.clarityelections.com/PA/Allegheny/63905/Web02.193333/#/cid/0104

**Comment by mixja** [comment]
This compares 2016 vs 2020 results for Allegheny county based upon above link for 2016 data - distributions look consistent to me:

<img width="2247" alt="2016 vs 2020" src="https://user-images.githubusercontent.com/3351083/98453133-b0df7400-21ba-11eb-8a57-c4e1b16ff048.png">

See https://github.com/mixja/2020_benfords

**Comment by iraykhel** [comment]
Do you mind running this comparison for Chicago and Milwaukee as well, if you have the data? I think this is probably the best claim against fraud.
